pkg/api/converterfromv20180930preview_test.go

```diff
@@ -34,15 +34,19 @@ func internalManagedCluster() *OpenShiftManagedCluster {
 	// this is the expected internal equivalent to
 	// v20180930previewManagedCluster()
 
+	planName := "Plan.Name"
+	planProduct := "Plan.Product"
+	planPromotionCode := "Plan.PromotionCode"
+	planPublisher := "Plan.Publisher"
 	return &OpenShiftManagedCluster{
 		ID:       "ID",
 		Location: "Location",
 		Name:     "Name",
 		Plan: ResourcePurchasePlan{
-			Name:          "Plan.Name",
-			Product:       "Plan.Product",
-			PromotionCode: "Plan.PromotionCode",
-			Publisher:     "Plan.Publisher",
+			Name:          &planName,
```

Comment:
use `to.StringPtr("Plan.Name")` here, etc.

Reply:
switched to the "to" package